nss-day-cohort-39/holiday-road-black-hole-badgers
# Added Font, Wording, Flexing, and a Title

### Details
- Chose 2 fonts to apply to the page
- Itinerary preview no longer expands with the saved itineraries
- Page given a title with an icon
- Shrunk the dropdowns

### How to Test
- Make sure main page features still work
- Make sure itinerary preview behaves how we originally imagined
